hummingbot/__init__.py

```diff
@@ -102,7 +102,10 @@ def add_remote_logger_handler(loggers):
         root_logger.error("Error adding remote log handler.", exc_info=True)
 
 
-def init_logging(conf_filename: str, override_log_level: Optional[str] = None, dev_mode: bool = False):
+def init_logging(conf_filename: str,
+                 override_log_level: Optional[str] = None,
+                 dev_mode: bool = False,
+                 strategy_file_path: str = "conf_unselected_strategy"):
```

Comment:
nitpick... how about making the default name not sound like it's a strategy? Something generic like "logs_hummingbot.log" (which means the suffix is "hummingbot") should do the job and cleanly separate the general log file from the strategy specific ones.

Reply:
@martinkou Sure. Changed.